java/Oleksii-Riabov/src/com/zoolatech/lecture3/tasks/_2/AccountValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.zoolatech.lecture3.tasks._2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AccountValidator {
+    public boolean isValid(UserAccount userAccount) {
+        List<AccountValidator> list = new ArrayList<>();
```

Comment:
It's fine to keep it as is, but since the class works with the same class, it might be beneficial to define the list and init it during the object construction